README.md

```diff
@@ -1,29 +1,61 @@
-# SJPG Ddecoder for LVGL
+# LV JPG / SJPG DECODER 
 
 # Overview:
-  - SJPG is a custom format based on "normal" JPG
-  - SJPG is 'split-jpeg' which is a bundle of small jpeg fragments with an sjpg header.
-  - SJPG size will be almost comparable to the jpg file or might be a slightly larger.
+  - lv_lib_splitjpg supports both normal jpg and the custom sjpg formats.
+  - Decoding normal jpg consumes RAM with the site fo the whole uncompressed image (recommended only for devices with more RAM)
+  - sjpg is a custom format based on "normal" JPG and specially made for lvgl.
+  - sjpg is 'split-jpeg' which is a bundle of small jpeg fragments with an sjpg header.
+  - sjpg size will be almost comparable to the jpg file or might be a slightly larger.
   - lv_sjpg can open and handle multiple files at same time.
   - File read from disk (fread) and c-array is implemented.
   - SJPEG frame fragment cache enables fast fetching of lines if availble in cache.
-  - By default the image cache will be xres * 2 * 16 bytes (can be modified)
+  - By default the sjpg image cache will be xres * 2 * 16 bytes (can be modified)
   - Currently only 16 bit image format is supported (to do)
 # Sample code:
-```c
-lv_obj_t * img1;
-lv_obj_t * img2;
-
-void lv_demo_sjpeg(void)
-{
-    lv_jpeg_init();
-    img1 = lv_img_create(lv_scr_act(), NULL);
-    img2 = lv_img_create(lv_scr_act(), NULL);
-    lv_img_set_src(img1,  &wallpaper);            //image 1 from c array
-    lv_img_set_src(img2,  ".\\small_image.sjpg")  //image 2 from file system (fopen)
-}
-```
+```c 
+    
+   /*-------
+   
+    USE lvgl/lv_fs_if driver for using lv_fs in simulator
+   
+   Modify lv_conf.h as below for testing in PC simulator
+    #define LV_USE_FS_IF	1
+    #if LV_USE_FS_IF
+    #  define LV_FS_IF_FATFS    '\0'
+    #  define LV_FS_IF_PC     'S'
+    #endif  
+  
+    For fast simulatneous multi jpg/sjpg rendering in expense of additional ram, 
+    you can modify LV_IMG_CACHE_DEF_SIZE to 2 or above if testing on simulator or
+    using it in devices like raspberry pi etc.
+  
+  -------*/
+   
+   LV_IMG_DECLARE(wallpaper);
+   LV_IMG_DECLARE(small_image);
+   LV_IMG_DECLARE(wallpaper_jpg);
+
+   //for PC simulator
+   lv_fs_if_pc_init();
+    
+   lv_obj_t * img1;
+   lv_obj_t * img2;
 
+   lv_split_jpeg_init();
+   img1 = lv_img_create(lv_scr_act(), NULL);
+   img2 = lv_img_create(lv_scr_act(), NULL);
+   
+   lv_img_set_src(img1,  &wallpaper_jpg);         //jpg image 1 from c array inside sample_sjpg_images
+   //lv_img_set_src(img1,  &wallpaper);            //sjpg image 1 from c array inside sample_sjpg_images
+   
+   lv_img_set_src(img2,  "S.\\lv_examples\\assets\\lv_lib_split_jpg\\sample_sjpg_images\\small_image.sjpg"); //sjpg file from file system ( using lv_fs )
```

Comment:
It'd be better to store the examples in this repo.

Reply:
This is done. Added this repo alongside with lvgl and added an example folder inside this repo with a very basic demo code using the images inside the example/images.